src/interaction/DihedralHarmonicNCos.hpp

```diff
@@ -202,7 +202,7 @@ namespace espressopp {
        * @return The value of the force
        */
       real _computeForceRaw(real phi) const {
-        return K * multiplicity * sin(phi0 - multiplicity*phi);
+        return -1.0*K * multiplicity * sin(phi0 - multiplicity*phi);
```

Comment:
I agree about the minus sign, to make it -gradV, but like I said above I wonder if there's also a 1/sin(phi) term.